reporter-web-app/src/models/WebAppOrtResult.js

```diff
@@ -454,6 +482,14 @@ class WebAppOrtResult {
         return false;
     }
 
+    hasDeclaredLicensesProcessed() {
+        if (this.#declaredLicensesProcessed.length > 0) {
```

Comment:
How about just: `return this.#declaredLicensesProcessed.length > 0` ?

..same below

Reply:
... and ideally also a follow-up commit to change this in existing code like `hasDetectedLicenses()`.

Reply:
Fixed it up in the same commit

Reply:
It's a minor change code-wise, but it's not related to the topic of "Remove unprocessed licenses from Summary" and thus should go to a separate commit.

Reply:
OK make it a separate commit as I found the same issue in multiple places